fe/fe-core/src/main/java/com/starrocks/lake/compaction/PartitionStatistics.java

```diff
@@ -32,6 +32,25 @@ public class PartitionStatistics {
     @SerializedName(value = "compactionScore")
     private Quantiles compactionScore;
 
+    // default priority is -1, manual compaction will have priority value 1
+    @SerializedName(value = "priority")
+    private volatile CompactionPriority priority = CompactionPriority.DEFAULT;
+
+    public enum CompactionPriority {
+        DEFAULT(-1),
+        MANUAL_COMPACT(1);
```

Comment:
```suggestion
        HIGH(1);
```

Reply:
`MANUAL_COMPACT` might be a better choice? will not modify it now